.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "submodules/ixdat-large-test-files"]
+	path = submodules/ixdat-large-test-files
+	url = git@github.com:ixdat/ixdat-large-test-files.git
```

Comment:
I think you will need to fix this up, to work of the https endpoint instead of ssh. Since the repo is public it should clone for everyone as https and this will required people to setup ssh, which we don't want to.